modules/subprocess/src/main/scala/notebook/kernel/pfork/BetterFork.scala

```diff
@@ -170,7 +169,7 @@ object BetterFork {
         } else {
           acc ++ (cl.asInstanceOf[URLClassLoader].getURLs map { u =>
             val f = new File(u.getFile)
-            URLDecoder.decode(f.getAbsolutePath, "UTF8")
+            URLDecoder.decode(f.getParent, "UTF8")
```

Comment:
yup, you know maybe we should give this a try: http://www.scala-sbt.org/sbt-native-packager/topics/longclasspath.html
I'll do it probably this week and keep you posted

thanks for the explanations